doc/releases/nautilus.rst

```diff
@@ -1,3 +1,176 @@
+v14.2.17 Nautilus
+=================
+
+This is the 17th backport release in the Nautilus series. We recommend
+users to update to this release.
+
+Notable Changes
+---------------
+
+* $pid expansion in config paths like ``admin_socket`` will now properly expand
+  to the daemon pid for commands like ``ceph-mds`` or ``ceph-osd``. Previously
+  only ``ceph-fuse``/``rbd-nbd`` expanded ``$pid`` with the actual daemon pid.
+* RADOS: PG removal has been optimized in this release.
+* RADOS: Memory allocations are tracked in finer detail in BlueStore and displayed as a part of the ``dump_mempools`` command.
+* RADOS: Several fixes in BlueStore to improve stability.
```

Comment:
```suggestion
* RADOS: Several fixes in BlueStore to improve stability.
* cephfs: clients which acquire capabilities too quickly are throttled to prevent instability. See new config option mds_session_cap_acquisition_throttle to control this behavior.
```